modules/Markbook/markbook_edit_copyProcess.php

```diff
@@ -63,7 +63,7 @@
 
                     //Write to database
                     try {
-                        $date = (!empty($_POST['date']))? dateConvert($guid, $_POST['date']) : date('Y-m-d');
```

Comment:
Same situation as my comment about line 42 of `markbook_edit_addMultiProcess.php`.